lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data[:from] && data[:from][:name]
```

Comment:
Also… from this code it looks like some videos might not have a Facebook page. 

If that's the case, you should write a test for that scenario.

In other words… why do you need to check if `data[:from]` is there?